voltaire_bundler/bundle/bundle_manager.py

```
@@ -1,14 +1,24 @@
 import asyncio
 import copy
+from collections import defaultdict
 from datetime import datetime
 import logging
 import math
 from typing import Any, cast
```

Flake8 (7.3.0)

[error] 'typing.Any' imported but unused

(F401)